<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>genericBinaryFileWidget</class>
 <widget class="QWidget" name="genericBinaryFileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>360</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="showFileListWdgt" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QVPushButton" name="showFileListQpb">
        <property name="text">
         <string>File list</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="showFileListVSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="fileListGBox">
      <property name="title">
       <string>File list</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <widget class="QWidget" name="fileListWdgt" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <property name="margin">
           <number>2</number>
          </property>
          <property name="spacing">
           <number>2</number>
          </property>
          <item row="1" column="3">
           <widget class="QPushButton" name="hideFileListQpb">
            <property name="minimumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ressources/genericBinaryFiles.qrc">
              <normaloff>:/img/Gnome-emblem-unreadable.png</normaloff>:/img/Gnome-emblem-unreadable.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="4">
           <widget class="FileList" name="fileList">
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
            <column>
             <property name="text">
              <string>File</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Type</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="removeFileQpb">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ressources/genericBinaryFiles.qrc">
              <normaloff>:/img/Gnome-user-trash.png</normaloff>:/img/Gnome-user-trash.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="openFileQpb">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ressources/genericBinaryFiles.qrc">
              <normaloff>:/img/Gnome-list-add.png</normaloff>:/img/Gnome-list-add.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="2" colspan="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="fileViewerTab">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileList</class>
   <extends>QTableWidget</extends>
   <header>filelist.h</header>
  </customwidget>
  <customwidget>
   <class>QVPushButton</class>
   <extends>QPushButton</extends>
   <header>qvpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ressources/genericBinaryFiles.qrc"/>
 </resources>
 <connections/>
</ui>
